datalad/crawler/pipelines/simple_with_standford_lib.py

```diff
@@ -0,0 +1,54 @@
+from ..nodes.matches import xpath_match
+try:
+    from scrapy.http import Response
+    from scrapy.http import XmlResponse
+    from scrapy.selector import Selector
+except ImportError:  # pragma: no cover
+    class Selector(object):
+        xpath = css = None
+    Response = None
+
+
+# copied the simple_with_archives template in order to show
+def pipeline_2(url=None,
+             x_pathmatch_='.*/download/.*\.(tgz|tar.*|xml)',
+             tarballs=True,
+             datalad_downloader=False,
+             use_current_dir=False,
+             leading_dirs_depth=1,
+             rename=None,
+             backend='MD5E',
+             add_archive_leading_dir=False,
+             annex=None,
+             incoming_pipeline=None):
+
+    crawler = crawl_xml()
```

Comment:
a regular crawl_url didn't work?  I think it has nothing HTML specific so should be capable to fetch and provide to the next nodes the xml content
just make above pipeline get the `doc_id` (not `url`), compose the url how you do in crawl_xml and it should work (or I could be wrong ;) ) if you provide it to crawl_url